README.md

```diff
@@ -90,7 +90,7 @@ Here is a list of the validators currently available.
 Validator                               | Description
 --------------------------------------- | --------------------------------------
 **contains(str, seed [, options ])**    | check if the string contains the seed.<br/><br/>`options` is an object that defaults to `{ ignoreCase: false, minOccurrences: 1 }`.<br />Options: <br/> `ignoreCase`: Ignore case when doing comparison, default false<br/>`minOccurences`: Minimum number of occurrences for the seed in the string. Defaults to 1.
-**equals(str, comparison)**             | check if the string matches the comparison.
+**equals(str, comparison [, options])**             | check if the string matches the comparison.<br/> `options` are optional and default they are defined to `{ sensitivity: undefined, locales: undefined }`.  <br/><br/> Options : <br/> `sensitivity` :  option is used to define the sensitivity option expected values `['base', 'variant', 'accent']` <br/> `locales` : option is used to define the local language . like `validator.equals('Hello','hello',{ sensitivity: 'accent', locales: 'en' }) // true`
```

Comment:
It would be good to refer to the [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) or something similar of `localeCompare` so people know better what the options are.

Reply:
And if we're using `localeCompare` I think it would be good to immediately support more options than just `sensitivity`